embulk-parser-json/build.gradle

```diff
@@ -62,6 +70,88 @@ javadocJar {
     from rootProject.file("LICENSE")
 }
 
+publishing {
+    publications {
+         maven(MavenPublication) {
+            afterEvaluate { project ->
+                groupId = "${project.group}"
+                artifactId = "${project.name}"
+            }
+
+            from components.java
+             // javadocJar and sourcesJar are added by java.withJavadocJar() and java.withSourcesJar() above.
+             // See: https://docs.gradle.org/current/javadoc/org/gradle/api/plugins/JavaPluginExtension.html
+
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                packaging "jar"
+
+                afterEvaluate { project ->
+                    name = "${project.name}"
+                    description = "${project.description}"
+                }
+                url = "https://www.embulk.org/"
+
+                licenses {
+                    license {
+                        // http://central.sonatype.org/pages/requirements.html#license-information
+                        name = "The Apache License, Version 2.0"
+                        url = "https://www.apache.org/licenses/LICENSE-2.0.txt"
+                    }
+                }
+
+                developers {
+                    developer {
+                        name = "Muga Nishizawa"
+                        email = "muga.nishizawa@gmail.com"
+                    }
+                    developer {
+                        name = "Masatoshi Shimada"
+                        email = "masatoshi.shimada@every.tv"
+                    }
```

Comment:
@smdmts We're going to separate `embulk-standards` plugins as individual plugins, then to release them as independent pure-Java Embulk plugin JARs into Maven Central. In this `embulk-parser-json`, we wanted to include you as a developer from the pull request #651. What do you think?

Reply:
@dmikurube Sorry for delay replying. I approve include my name and appreciate it. Thank you.